front/app/containers/Admin/projects/edit/messages.ts

```diff
@@ -103,7 +103,7 @@ export default defineMessages({
   createSurveyDescription: {
     id: 'app.containers.AdminPage.ProjectEdit.createSurveyDescription',
     defaultMessage:
-      'Embed a Typeform, Google Form, Enalyzer, SurveyXact or Qualtrics survey.',
+      'Embed a Typeform, Google Form, Enalyzer, SurveyXact, Qualtrics or Microsoft Forms survey.',
```

Comment:
This won't be registered if I recall correctly and it didn't change, product needs to change through crowdin (new keys get pushed to crowdin, creating the .json files, which then get fully managed from the crowdin ecosystem with more of a push mechanism back to our repo, so I think they don't catch changing base messages). Might be mistaken, Brent's the expert there.

Reply:
Thanks for pointing it out! Will mention to Karel that it needs changing :)